io.openems.edge.meter.eastron/src/io/openems/edge/meter/eastron/sdm120/MeterEastronSdm120Impl.java

```diff
@@ -0,0 +1,223 @@
+						m(new FloatDoublewordElement(30001 - offset)) //
+								.m(ElectricityMeter.ChannelId.VOLTAGE, SCALE_FACTOR_3) //
+								.m(ElectricityMeter.ChannelId.VOLTAGE_L1, chain(//
+										SCALE_FACTOR_3, //
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L1))) //
+								.m(ElectricityMeter.ChannelId.VOLTAGE_L2, chain(//
+										SCALE_FACTOR_3, //
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L2))) //
+								.m(ElectricityMeter.ChannelId.VOLTAGE_L3, chain(//
+										SCALE_FACTOR_3, //
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L3))) //
+								.build(), //
+						new DummyRegisterElement(30003 - offset, 30006 - offset), //
+						m(new FloatDoublewordElement(30007 - offset)) //
+								.m(ElectricityMeter.ChannelId.CURRENT, SCALE_FACTOR_3) //
+								.m(ElectricityMeter.ChannelId.CURRENT_L1,
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L1)) //
+								.m(ElectricityMeter.ChannelId.CURRENT_L2,
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L2)) //
+								.m(ElectricityMeter.ChannelId.CURRENT_L3,
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L3)) //
+								.build(), //
+						new DummyRegisterElement(30009 - offset, 30012 - offset), //
+						m(new FloatDoublewordElement(30013 - offset)) //
+								.m(ElectricityMeter.ChannelId.ACTIVE_POWER, INVERT_IF_TRUE(this.invert)) //
+								.m(ElectricityMeter.ChannelId.ACTIVE_POWER_L1, chain(//
+										INVERT_IF_TRUE(this.invert),
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L1))) //
+								.m(ElectricityMeter.ChannelId.ACTIVE_POWER_L2, chain(//
+										INVERT_IF_TRUE(this.invert),
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L2))) //
+								.m(ElectricityMeter.ChannelId.ACTIVE_POWER_L3, chain(//
+										INVERT_IF_TRUE(this.invert),
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L3))) //
+								.build(), //
+						new DummyRegisterElement(30015 - offset, 30024 - offset), //
+						m(new FloatDoublewordElement(30025 - offset)) //
+								.m(ElectricityMeter.ChannelId.REACTIVE_POWER, INVERT_IF_TRUE(this.invert)) //
+								.m(ElectricityMeter.ChannelId.REACTIVE_POWER_L1, chain(//
+										INVERT_IF_TRUE(this.invert),
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L1))) //
+								.m(ElectricityMeter.ChannelId.REACTIVE_POWER_L2, chain(//
+										INVERT_IF_TRUE(this.invert),
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L2))) //
+								.m(ElectricityMeter.ChannelId.REACTIVE_POWER_L3, chain(//
+										INVERT_IF_TRUE(this.invert),
+										SET_ZERO_IF_TRUE(this.config.phase() != SinglePhase.L3))) //
+								.build(),
+						new DummyRegisterElement(30027 - offset, 30070 - offset), //
+						m(ElectricityMeter.ChannelId.FREQUENCY, new FloatDoublewordElement(30071 - offset) //
+								.wordOrder(WordOrder.MSWLSW).byteOrder(ByteOrder.BIG_ENDIAN), SCALE_FACTOR_3)));
```

Comment:
`WordOrder.MSWLSW` and `ByteOrder.BIG_ENDIAN` are already the defaults. I am removing this.